FirebaseAuthInteroperable.podspec

```diff
@@ -23,5 +23,5 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = '10.10'
   s.tvos.deployment_target = '10.0'
   s.source_files = 'Interoperability/Auth/**/*.h'
-  s.public_header_files = 'Interoperability/Auth/Public/*.h'
```

Comment:
I believe you also need:

`s.public_header_files = 'Interoperability/Auth/Private/*.h'`